internal/endpoints/awsrulesfn/generate.go

```diff
@@ -0,0 +1,7 @@
+//go:build codegen
+// +build codegen
+
+package awsrulesfn
+
+//go:generate go run -tags codegen ./internal/partition/codegen.go -model partitions.json -output partitions.go
```

Comment:
**Question/Correctness**: Is this actually used? I'm assuming this is related to implementing the `partition` function but I'm not seeing how this is wired together actually and what it generates.

Reply:
gotcha, yah i agree. will remove.